### ISSUE TITLE
Ruby is not an appropriate mechanism for managing speech data

### ISSUE THREAD

**Issue by r12a** [w3c/pronunciation#47]
7. Use Case Ruby
https://www.w3.org/TR/pronunciation-use-cases/#use-case-ruby

> ruby guides pronunciation visually. This seems like a natural fit for text-to-speech.

As mentioned in the text, ruby markup is primarily a way of **visually** aligning annotations with base text, and the standard usage is for both annotation and base to be presented to the user and to be present in element content. However, not all ruby annotations are associated with pronunciation (eg. see https://w3c.github.io/clreq/#h-indicating_meaning_or_other_information).  

Although ruby annotation can indicate pronunciation, the i18n WG does not see this as a natural fit for general text-to-speech semantics, and recommends against ruby being considered as _the_ format to use for expressing pronunciation information.

If ruby information is present, however, **and** it is known to contain pronunciation information, it _may_ be possible for a speech processor to extract **some** pronunciation information from that markup. Because the ruby annotation may not contain pronunciation information, or if it does, will usually present it using kana, bopomofo, or a form of pinyin, some special treatment is necessary.

**Comment by murata2makoto** [comment]
I agree with @r1a.  Ruby text typically consists of hiragana or katakana characters rather than CJK ideographic characters.  If T2S engines use ruby text rather than base characters, morphological analysis is likely to fail.  Such failures willl lead to unnatural pronunciation.

**Comment by murata2makoto** [comment]
Having compared other implementation approaches and ruby, I think that the subclause "Use Case Ruby" should be dropped or converted to a description of misuse.

**Comment by murata2makoto** [comment]
Although I proposed to delete "Use Case Ruby", I propose to add "Abuse of Ruby" and explicitly mention 

>AT "double reading" words instead of choosing either the content or the annotation"

since such "double reading" is extremely harmful.
